### PR TITLE
did some refactoring

### PR DESCRIPTION
**Pull request by Izzycodez**

### CODE REVIEW

**Review by mykoman — APPROVED** [review]

File: app/component/Footer.tsx [hunk 1]
```diff
@@ -2,8 +2,11 @@ import React from "react";
 
 export default function Footer() {
   return (
-    <footer className="text-gray-400 text-center text-sm -y-5">
-      <p>Copyright &copy; {new Date().getFullYear()}</p>
+    <footer
+      style={{ minHeight: "6vh" }}
```
Comment (WajuAbolarin) [inline app/component/Footer.tsx:6]: Why are you mixing `style` with `className`,you can add these `6vh` min-height with a class.

File: app/component/Nav.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { usePathname } from "next/navigation";
 import { useEffect, useState } from "react";
 
 const Nav = () => {
-  const [accessToken, setAccessToken] = useState<string>("");
+  const [accessToken, setAccessToken] = useState<string | null>("");
```
Comment (WajuAbolarin) [inline app/component/Nav.tsx:7]: if you want this to be nullable the default should be `null` not `""`

File: app/component/Nav.tsx [hunk 3]
```diff
@@ -19,30 +19,34 @@ const Nav = () => {
...
     setTimeout(() => {
       window.location.href = "/";
-    }, 5000);
+    }, 200);
   };
 
   return (
-    <header className="text-white-200 w-full bg-gray-500">
+    <header
+      style={{ minHeight: "8vh" }}
```
Comment (WajuAbolarin) [inline app/component/Nav.tsx:47]: same here

File: app/component/Footer.tsx (hunk 1 above)
Comment (Izzycodez) [reply app/component/Footer.tsx:6]: You mean i should make a custom class for it?

File: app/component/Nav.tsx (hunk 2 above)
Comment (Izzycodez) [reply app/component/Nav.tsx:7]: okay sir

File: app/component/Nav.tsx (hunk 3 above)
Comment (Izzycodez) [reply app/component/Nav.tsx:47]: okay sir

File: app/component/Footer.tsx (hunk 1 above)
Comment (WajuAbolarin) [reply app/component/Footer.tsx:6]: no, use tailwind.https://tailwindcss.com/docs/adding-custom-styles

Comment (Izzycodez) [reply app/component/Footer.tsx:6]: Ok, I get now.

**Pull request merged**